src/Site.js

```diff
@@ -666,32 +670,77 @@ function getPluginPath(rootPath, plugin) {
   }
 
   // Check in src folder
-  const defaultPath = path.join(__dirname, BUILT_IN_PLUGIN_FOLDER_NAME, `${plugin}.js`);
+  const srcPath = path.join(__dirname, BUILT_IN_PLUGIN_FOLDER_NAME, `${plugin}.js`);
+  if (fs.existsSync(srcPath)) {
+    return srcPath;
+  }
+
+  // Check in default folder
+  const defaultPath = path.join(__dirname, BUILT_IN_DEFAULT_PLUGIN_FOLDER_NAME, `${plugin}.js`);
   if (fs.existsSync(defaultPath)) {
     return defaultPath;
   }
 
   return '';
 }
 
+/**
+ * Finds plugins in the site's default plugin folder
+ */
+function findDefaultPlugins() {
+  const globPath = path.join(__dirname, BUILT_IN_DEFAULT_PLUGIN_FOLDER_NAME);
+  if (!fs.existsSync(globPath)) {
+    return [];
+  }
+  return walkSync(globPath, {
+    directories: false,
+    globs: ['*.js'],
```

Comment:
A possible concern with this could be that a `.js` files happens to have a `preRender()` and/or `postRender()` method, but it is not an actual plugin, but just some library.

Should we have some way to allow .js files to "designate" themselves as a valid MarkBind plugin?

Reply:
Maybe we can use a different file extension, like .mbjs or something? Just for default built-in plugins, the other plugins can be .js since those are only loaded when specified

Reply:
Hmm, I am not sure whether it is a good idea to give a different file extension. They are still essentially normal JavaScript files, and that is not going to play well with editor configurations...

Perhaps introduce a suffix instead to mark such scripts? E.g. `<name>-mbplugin.js`.

Reply:
I suggest using a prefix, similar to ESLint.

Reply:
i.e. `markbind-plugin-<name>.js`